controllers/Category.js

```diff
@@ -24,7 +24,11 @@ exports.getOneCategory = (req, res) => {
         catNotFound(res, id)
       }
 
-      Package.find({ categories: id }).then(packages => {
+      Package.find({
+        categories: {
```

Comment:
This is not needed here.
The previous syntax would work. 
$in is only needed if you are giving an array.
But here `id` is just a single id so $in is not necessary.